src/utils/localFormatted/localFormatted.ts

```diff
@@ -1,47 +1,35 @@
-export const locale =
-  (navigator.languages && navigator.languages[0]) ||
-  navigator.language ||
-  'en-US';
+export function formattedNumber(
+  value: any,
```

Comment:
nor `any`, nor `string`, no implicits, we should be strict here
only a `number` type is possible

see original `toLocaleString` impl for the reference